json_logging/__init__.py

```diff
@@ -31,6 +31,10 @@
 _request_util = None
 _default_formatter = None
 
+def request_util():
```

Comment:
Can u add comment to state that its is not recommended to request_util directly

Reply:
How about changing `BaseJSONFormatter._format_log_object` to a Public/Exported methods, so I can extends from `BaseJSONFormatter` instead of `logging.Formatter`?

Reply:
yeah, that will be a better approach 👍